src/components/dev-hub/stories/badge.stories.mdx

```diff
@@ -0,0 +1,53 @@
+import { Meta, Story, Preview } from '@storybook/addon-docs/blocks';
+import Badge from '../badge';
+import { colorMap } from '../theme';
+
+<Meta title="Badge" />
+
+# Badge
+
+Badges can be used to label contents. It accepts two parameters: `contentType` and `color`.
+* The `contentType` variable defines the text that goes inside the badge.
+* The optional `color` variable defines the color of the borders around the badge.
+
+<Preview>
+    <Story name="custom-badge">
+        <div style={{position:'relative', height: '50px'}}>
+            <Badge contentType={'badge'} color={colorMap.teal}/>
```

Comment:
I noticed the font is off a bit but this is likely something with the storybook configuration. Don't sweat this I will look into it more separately.